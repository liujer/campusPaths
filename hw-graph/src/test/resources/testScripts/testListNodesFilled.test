# Tests the method listNodes in DLGraph by adding
# nodes to graphs and checking if nodes are outputted
# correctly
CreateGraph A
AddNode A one
AddNode A two
ListNodes A