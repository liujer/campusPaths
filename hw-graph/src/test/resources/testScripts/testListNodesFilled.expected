# Tests the method listNodes in DLGraph by adding
# nodes to graphs and checking if nodes are outputted
# correctly
CreateGraph A
Created graph A
Added node one to A
Added node two to A
ListNodes of A output: one two